public/wcExternalDownload.html

```diff
@@ -0,0 +1,42 @@
+<!-- Run yarn build, to test if app still works as webcomponent -->
+<html>
+  <head>
+    <script>
+      window.changeAttr = function (attr, val) {
+        // If attribute value is the same the update is not triggered,
+        // so we set it to null first before setting it to the new value.
+        var map = document.getElementById('map');
+        // On IE only lowercase attribute are detected.
+        map.setAttribute(attr.toLowerCase(), val);
+      };
+      fetch('https://backend.developer.geops.io/publickey')
+        .then(function (response) {
+          return response.json();
+        })
+        .then(function (data) {
+          changeAttr('apiKey', data.key);
+        })
+        .catch(function () {
+          console.error('Request to get the apiKey failed');
+        });
+    </script>
+  </head>
+  <body>
+    <div style="width: 800px; height: 500px">
+      <!-- you must specify an api key to make it work -->
+      <trafimage-maps
+        id="map"
+        activeTopicKey="ch.railplus.mitglieder"
+      ></trafimage-maps>
+      <button onclick="downloadPdf()">Download PDF</button>
+    </div>
+    <!-- <script src="./build/bundle.js"></script> -->
+    <script src="https://unpkg.com/trafimage-maps@1.21.22-beta.5/bundle.js"></script>
+    <script>
+      function downloadPdf() {
+        const tm = document.getElementsByTagName('trafimage-maps')?.[0];
+        tm?.dispatchEvent(new Event('pdf-download'));
+      }
```

Comment:
why not using the index.html page for this ? you can provide a topic in the url... I think

Reply:
I created an input field in the development box in index.html where you can post your message